web/app/Repository/GameRepository.php

```diff
@@ -71,6 +59,12 @@ public function storeGame(array $post): bool
         $id_white = $post["player_white"];
         $id_black = $post["player_black"];
 
+        echo "<h1>"."Spiel wurde eingetragen"."</h1>";
```

Comment:
pro tip (many php devs to not know that):
instead of concatenating strings just to echo it, you can echo multiple strings in a single command with a comma:
echo '<h1>', 'Spiel wurde eingetragen', '</h1>';
in this specific case, you can even put everything together. there is no need to separate html from content.
echo '<h1>Spiel wurde eingetragen</h1>';

Reply:
Yes i know but i want it to be overseeable